packages/sync/src/config/AuthContextProvider.ts

```diff
@@ -1,7 +1,7 @@
 
 export interface AuthContext {
   header: any;
-  token: string;
+  token?: string;
```

Comment:
Why token is optional? :>

Reply:
No that was a mistake, pushing changes to check token before resolving